fastp/0.23.2/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:bionic as app
+
+ARG FASTP_VER="0.23.2"
+
+LABEL base.image="ubuntu:bionic"
+LABEL dockerfile.version="1"
+LABEL software="Fastp"
+LABEL software.version=$SOFTWARENAME_VER
+LABEL description="An ultra-fast all-in-one FASTQ preprocessor"
+LABEL website="https://github.com/OpenGene/fastp"
+LABEL license="https://github.com/OpenGene/fastp/blob/master/LICENSE"
+LABEL maintainer="Idowu Olawoye"
+LABEL maintainer.email="olawoyei0303@run.edu.ng"
+
+RUN apt-get update && apt-get install -y \
+ wget
```

Comment:
Could you also include `apt-get autoclean` and `rm -rf /var/lib/apt/lists/*`? This helps reduce the container size. Also, (while I'm remembering) could you add `ca-certificates`? This helps those that are using these containers with nextflow tower.
```
RUN apt-get update && apt-get install -y \
 wget \
 ca-certificates && \
 apt-get autoclean && rm -rf /var/lib/apt/lists/*
 ```